### PR TITLE
Japanese Bible lessons, Incomplete verse parser, Updated README

### PR DESCRIPTION
**Pull request by imasaru**
Whew, I finally did it. Well... _most of it_. I want to help with finishing the verse parser, but I remember very little JavaScript...

### CODE REVIEW

**Comment by imasaru** [comment]
What is going on with Travis? He seems to have personal issues with me. lol

OpenSSL is giving an error. Is it with signing the .apk upon being built or something?

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Below is current list of languages and capabilities implemented
 |🇪🇸 🇲🇽 Spanish|✅ Yes|✅ Yes|❌ No|❌ No|
 |🇹🇷 Turkish|✅ Yes|❌ No|❌ No|❌ No|
 |🇺🇦 Ukrainian|✅ Yes|✅ Yes|✅ Yes|❌ No|
+|🇯🇵 Japanese|✅ Yes|❌ No|❌ No|❌ No|
```
Comment (imasaru) [inline README.md]: Well, maybe adding this line before the Japanese update is a bad idea... Hold off on this commit until later?

**Comment by VitalikL** [comment]
@imasaru 
Few things that need modification
- [ ] Rename the root of the directory from `2017-1` to `2017-01`
- [ ] Populate all info.yml files with titles for the week. This will be displayed when quarterly is opened. 
- [ ] Undefined is better to be replaced with placeholders

**Comment by imasaru** [comment]
Well, the thing is, the Japanese Conference doesn't publish the lessons ahead of time apparently, and their website only lets you download the current week's lesson, so therefore I cannot know the weekly lesson titles and am unable to include them in the info.yml files. I'll e-mail the Conference again to see if they can give me a download of the future lessons. 

Also, the creation script hasn't been updated to include Japanese placeholders, which is why the empty lessons are "undefined," right? I'll take the Japanese translations from POEditor and edit the script to include them myself, if that will make things easier.

**Comment by VitalikL** [comment]
Good job @imasaru. Please rebase to make the PR target `stage` branch

**Pull request closed without merging**